stem/IO/kratos_boundaries_io.py

```diff
@@ -0,0 +1,156 @@
+            "process_name": "ApplyVectorConstraintTableProcess",
+            "Parameters": deepcopy(boundary.boundary_parameters.__dict__),
+        }
+
+        boundary_dict["Parameters"][
+            "model_part_name"
+        ] = f"{self.domain}.{boundary.part_name}"
+        boundary_dict["Parameters"]["variable_name"] = "DISPLACEMENT"
+        boundary_dict["Parameters"]["table"] = [0, 0, 0]
+
+        return boundary_dict
+
+    def __create_rotation_constraint_dict(self, boundary: Boundary) -> Dict[str, Any]:
+        """
+        Creates a dictionary containing the rotation constraint parameters
+
+        Args:
+            boundary (Boundary): rotation constraint object
+
+        Returns:
+            Dict[str, Any]: dictionary containing the boundary parameters
+        """
+
+        # initialize boundary dictionary
+        boundary_dict: Dict[str, Any] = {
+            "python_module": "apply_vector_constraint_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name": "ApplyVectorConstraintTableProcess",
+            "Parameters": deepcopy(boundary.boundary_parameters.__dict__),
+        }
+
+        boundary_dict["Parameters"][
+            "model_part_name"
+        ] = f"{self.domain}.{boundary.part_name}"
+        boundary_dict["Parameters"]["variable_name"] = "ROTATION"
+        boundary_dict["Parameters"]["table"] = [0, 0, 0]
+
+        return boundary_dict
+
+    def __create_absorbing_boundary_dict(self, boundary: Boundary) -> Dict[str, Any]:
+        """
+        Creates a dictionary containing the absorbing boundary parameters
+
+        Args:
+            boundary (Boundary): absorbing boundary object
+
+        Returns:
+            Dict[str, Any]: dictionary containing the boundary parameters
+        """
+
+        # initialize boundary dictionary
+        boundary_dict: Dict[str, Any] = {
+            "python_module": "set_absorbing_boundary_parameters_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name": "SetAbsorbingBoundaryParametersProcess",
+            "Parameters": boundary.boundary_parameters.__dict__,
+        }
+
+        boundary_dict["Parameters"][
+            "model_part_name"
+        ] = f"{self.domain}.{boundary.part_name}"
+
+        return boundary_dict
+
+    def __create_boundary_dict(self, boundary: Boundary) -> Dict[str, Any]:
+        """
+        Creates a dictionary containing the boundary parameters
+
+        Args:
+            boundary (Load): boundary object
+
+        Returns:
+            Dict[str, Any]: dictionary containing the boundary parameters
+        """
+
+        # add boundary parameters to dictionary based on boundary type.
+
+        if isinstance(boundary.boundary_parameters, DisplacementConstraint):
+            return self.__create_displacement_constraint_dict(boundary=boundary)
```

Comment:
```suggestion
            return self.__create_displacement_constraint_dict(boundary)
```

same for the others

Reply:
Currently changed due to change in signature of the inputs (part_name, boundary_parameters).
`boundary` is now used as self.__create_displacement_constraint_dict(**boundary.__dict__).